lib/apiary/command/preview.rb

```diff
@@ -30,11 +30,15 @@ def initialize(opts)
         @options = OpenStruct.new(opts)
         @options.path         ||= 'apiary.apib'
         @options.api_host     ||= 'api.apiary.io'
-        @options.headers      ||= {:accept => 'text/html', :content_type => 'text/plain'}
         @options.port         ||= 8080
         @options.proxy        ||= ENV['http_proxy']
         @options.server       ||= false
-	@options.host	      ||= '127.0.0.1'
+	      @options.host         ||= '127.0.0.1'
```

Comment:
yes, but in editor is ok.

![image](https://cloud.githubusercontent.com/assets/100356/14684579/efe71292-0730-11e6-9d7c-2bb1fa4099db.png)